contracts/src/ProtocolAdapter.sol

```diff
@@ -285,20 +292,21 @@ contract ProtocolAdapter is
 
         bytes32 logicRef = input.verifyingKey;
 
+        // The logic hash (availiable in the compliance proof) and the logic verifying key are assumed to be the same
+        // in this implementation.
```

Comment:
Let's avoid introducing the new term "logic hash". We called it logic reference throughout this codebase